apps/site/assets/js/algolia-autocomplete-with-geo.js

```diff
@@ -54,26 +53,6 @@ class AlgoliaAutocompleteWithGeo extends AlgoliaAutocomplete {
     this._indices.push("popular");
   }
 
-  bind() {
-    super.bind();
-    this.onFocus = this.onFocus.bind(this);
-  }
-
-  _addListeners() {
-    super._addListeners();
```

Comment:
Are we sure removing this and the above `super.bind()` aren't going to cause an adverse side effect?